demisto_sdk/commands/generate_docs/tests/generate_docs_test.py

```diff
@@ -498,13 +498,41 @@ def teardown_class(cls):
         cls.rm_readme()
 
     def test_generate_integration_doc(self):
+        """
+        Given
+            - generate_table_section command
+            - integration
+        When
+            - running generate_integration_doc command on the integration, generating setup section
+        Then
+            - Validate that the generating setup was created correctly, the numbers in the list are not being reset,
+             after a list item that contains a table
+            - Test that the predefined values and default values are added to the README.
```

Comment:
```suggestion
            - YML file representing an integration.
        When
            - Running generate_integration_doc command on the integration.
        Then
            - Validate that the generating setup was created correctly, specifically that the numbers in the lines are not being reset after a list item that contains a table.
            - Test that the predefined values and default values are added to the README.
```
Why did you write in the when section `generating setup section`?

Reply:
copy paste mistake